CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 See the [releases](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases) for details on bug fixes and added features.
 
+7.7.0
+=====
+### CVE package updates
+[CVE-2024-30105](https://github.com/advisories/GHSA-hh2w-p6rv-4g7w)
+
+- A derived `ClaimsIdentity` where claim retrieval is case-sensitive. The current `ClaimsIdentity`, in .NET, retrieves claims in a case-insensitive manner which is different than querying the underlying `SecurityToken`. The new `CaseSensitiveClaimsIdentity` class provides consistent retrieval logic with `SecurityToken`. Opt in to the new behavior via an AppContext switch. See PR [#2710](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2710) for details.
```

Comment:
2710 is for 6x

Reply:
yes, I didn't see one for 7x yet...more of a placeholder for now.

Reply:
updated